tests/unit/single_target_shortest_path.c

```diff
@@ -26,11 +26,10 @@
 #include "test_utilities.inc"
 
 void igraph_warnings_ignore(const char *reason, const char *file,
```

Comment:
We already have `igraph_warning_handler_ignore()`. Can such tests simply use it instead of providing their own?

Reply:
Yes. I also looked for other cases, but those were printing handlers, and those made sense because igraph's own handlers print line numbers, which is not nice in tests.